<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IChanConfig</class>
 <widget class="QDialog" name="IChanConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>8</y>
     <width>418</width>
     <height>285</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_general">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Channel modes</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QCheckBox" name="mode_n">
          <property name="toolTip">
           <string>Makes it impossible for people not member of the channel to send /msg to this channel</string>
          </property>
          <property name="text">
           <string>No external messages (n)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="mode_k">
          <property name="statusTip">
           <string>To join this channel a key has to be used. /join #Channel yourkey</string>
          </property>
          <property name="text">
           <string>Channel key (k)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLineEdit" name="mode_k_val">
          <property name="statusTip">
           <string>Set your channel key here</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="mode_t">
          <property name="toolTip">
           <string>Topic can only be changed by channel operators</string>
          </property>
          <property name="text">
           <string>Only op can set topic (t)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QCheckBox" name="mode_l">
          <property name="statusTip">
           <string>Maximum members this channel can hold</string>
          </property>
          <property name="text">
           <string>Channel limit (l)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="mode_l_val">
          <property name="statusTip">
           <string>Set the maximum number of members here</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="mode_i">
          <property name="toolTip">
           <string>Members must be invited first, /invite nickname #channel</string>
          </property>
          <property name="text">
           <string>Invite only (i)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="mode_p">
          <property name="toolTip">
           <string>Hide the channel from /list, /whois, etc.</string>
          </property>
          <property name="text">
           <string>Private (p)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="mode_s">
          <property name="statusTip">
           <string>Hide the channel from /list, /whois, etc, and server act like this channel doesn't even exist</string>
          </property>
          <property name="text">
           <string>Secret (s)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="mode_m">
          <property name="statusTip">
           <string>Prevents &quot;regular&quot; members to talk. Only members with voice and op can talk.</string>
          </property>
          <property name="text">
           <string>Moderated (m)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="edTopic"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Topic</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QFrame" name="frame">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>295</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnSave">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_ban">
    <attribute name="title">
     <string>Bans</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0" rowspan="4">
      <widget class="QTableView" name="banView">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="banAdd">
       <property name="text">
        <string>A</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/gfx/add.png</normaloff>:/icons/gfx/add.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="banDel">
       <property name="text">
        <string>R</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/gfx/remove.png</normaloff>:/icons/gfx/remove.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="banEdit">
       <property name="text">
        <string>E</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/gfx/edit.png</normaloff>:/icons/gfx/edit.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>183</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_exception">
    <attribute name="title">
     <string>Ban exceptions</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0" rowspan="4">
      <widget class="QTableView" name="exceptionView">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="exceptionAdd">
       <property name="text">
        <string>A</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/gfx/add.png</normaloff>:/icons/gfx/add.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="exceptionDel">
       <property name="text">
        <string>R</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/gfx/remove.png</normaloff>:/icons/gfx/remove.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="exceptionEdit">
       <property name="text">
        <string>E</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/gfx/edit.png</normaloff>:/icons/gfx/edit.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>183</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_invite">
    <attribute name="title">
     <string>Invite list</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0" rowspan="4">
      <widget class="QTableView" name="inviteView">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="inviteAdd">
       <property name="text">
        <string>A</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/gfx/add.png</normaloff>:/icons/gfx/add.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="inviteDel">
       <property name="text">
        <string>R</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/gfx/remove.png</normaloff>:/icons/gfx/remove.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="inviteEdit">
       <property name="text">
        <string>E</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/gfx/edit.png</normaloff>:/icons/gfx/edit.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>183</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>298</y>
     <width>87</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="close_button">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close_button</sender>
   <signal>clicked()</signal>
   <receiver>IChanConfig</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
